contrib/ffmpeg/module.defs

```diff
@@ -145,14 +145,17 @@ ifeq (1,$(FEATURE.qsv))
 endif
 
 ifeq (1,$(FEATURE.vce))
-  FFMPEG.CONFIGURE.extra += --enable-amf --enable-encoder=h264_amf --enable-encoder=hevc_amf 
+  FFMPEG.CONFIGURE.extra += --enable-amf --enable-encoder=h264_amf --enable-encoder=hevc_amf
 endif
 
 ifeq (1,$(FEATURE.nvenc))
 FFMPEG.CONFIGURE.extra += \
      --enable-ffnvcodec \
      --enable-encoder=h264_nvenc \
-     --enable-encoder=hevc_nvenc
+     --enable-encoder=hevc_nvenc \
+     --enable-nonfree \
```

Comment:
Something to note, "--enable-nonfree" isn't GPLv2 compatible, so this isn't something we can have turned on as it would violate the ffmpeg and our own license if we did. 
I was under the impression that with the nvcodec headers, we shouldn't need this